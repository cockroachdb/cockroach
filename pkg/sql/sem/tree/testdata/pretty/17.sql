select * from a order by index abc2@ba desc, INDEX abc2@ba, PRIMARY KEY abc2 DESC, "foo.bar" ASC
