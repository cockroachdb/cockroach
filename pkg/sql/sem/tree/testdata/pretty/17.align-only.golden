// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	*
FROM
	a
ORDER BY
	INDEX abc2@ba
		DESC,
	INDEX abc2@ba,
	PRIMARY KEY abc2
		DESC,
	"foo.bar"
		ASC

10:
----------
  SELECT *
    FROM a
ORDER BY INDEX abc2@ba
			DESC,
         INDEX abc2@ba,
         PRIMARY KEY abc2
			DESC,
         "foo.bar"
			ASC

22:
----------------------
  SELECT *
    FROM a
ORDER BY INDEX abc2@ba
			DESC,
         INDEX abc2@ba,
         PRIMARY KEY abc2
			DESC,
         "foo.bar" ASC

28:
----------------------------
  SELECT *
    FROM a
ORDER BY INDEX abc2@ba DESC,
         INDEX abc2@ba,
         PRIMARY KEY abc2
			DESC,
         "foo.bar" ASC

31:
-------------------------------
  SELECT *
    FROM a
ORDER BY INDEX abc2@ba DESC,
         INDEX abc2@ba,
         PRIMARY KEY abc2 DESC,
         "foo.bar" ASC

80:
--------------------------------------------------------------------------------
  SELECT *
    FROM a
ORDER BY INDEX abc2@ba DESC, INDEX abc2@ba, PRIMARY KEY abc2 DESC, "foo.bar" ASC

96:
------------------------------------------------------------------------------------------------
SELECT * FROM a ORDER BY INDEX abc2@ba DESC, INDEX abc2@ba, PRIMARY KEY abc2 DESC, "foo.bar" ASC


